shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -735,6 +753,14 @@ private void ensureAlive() {
     @NonNull
     Context getContext();
 
+    /**
+     * Returns true if the {@link FlutterActivityAndFragmentDelegate} should send the deeplinking
+     * URL to the framework through the {@code NavigationChannel.setInitialRoute} or {@code
+     * NavigationChannel.pushRoute}.
+     */
+    @Nullable
+    boolean shouldHandleDeeplinking();
```

Comment:
note the classes on flutteractivity/fragment are the user facing docs. This interface is package private.